DB-schema-migration.md

```diff
@@ -0,0 +1,156 @@
+# DB Schema migration for module update
+
+Copyright (C) 2016-2019 The Open Library Foundation
```

Comment:
Can you remove the Copyright line? The line in the README.md covers the whole repository and is the single place where we want to update the year.
See https://dev.folio.org/guidelines/contributing/#no-license-header

Reply:
Done